repose-aggregator/services/atom-feed/impl/src/main/resources/META-INF/schema/config/atom-feed-service.xsd

```diff
@@ -0,0 +1,183 @@
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+
+        <xs:attribute name="entry-order" type="EntryOrderType" use="optional" default="update">
+            <xs:annotation>
+                <xs:documentation>
+                    <html:p>
+                        Defines the order in which new Atom events will be delivered to any listeners registered
+                        on this feed.
+                    </html:p>
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+    </xs:complexType>
+
+    <xs:simpleType name="EntryOrderType">
+        <xs:restriction base="xs:string">
+            <xs:enumeration value="random">
+                <xs:annotation>
+                    <xs:documentation>
+                        <html:p>
+                            With this ordering, a feed listener will receive events with no ordering guarantees.
+                        </html:p>
+                    </xs:documentation>
+                </xs:annotation>
+            </xs:enumeration>
+
+            <xs:enumeration value="update">
+                <xs:annotation>
+                    <xs:documentation>
+                        <html:p>
+                            With this ordering, a feed listener will receive events in the order in which they were
+                            updated. That is, entries will be ordered chronologically based on their updated field
+                            from oldest to newest.
+                        </html:p>
+                    </xs:documentation>
+                </xs:annotation>
+            </xs:enumeration>
+
+            <!--xs:enumeration value="publish">
+                <xs:annotation>
+                    <xs:documentation>
+                        <html:p>
+                            With this ordering, a feed listener will receive events in the order in which they were
+                            published.
+                        </html:p>
+                    </xs:documentation>
+                </xs:annotation>
+            </xs:enumeration-->
+        </xs:restriction>
+    </xs:simpleType>
+
+    <xs:simpleType name="PositiveIntType">
+        <xs:restriction base="xs:int">
+            <xs:minInclusive value="1"/>
+        </xs:restriction>
+    </xs:simpleType>
+
+    <xs:complexType name="AuthenticationType">
+        <xs:attribute name="fqcn" type="xs:string"
+                      default="org.openrepose.nodeservice.atomfeed.impl.auth.NoopAuthenticatedRequestFactory">
+            <xs:annotation>
+                <xs:documentation>
+                    <html:p>Defines the fully qualified class name of a class which can authenticate requests.</html:p>
+                </xs:documentation>
+            </xs:annotation>
+        </xs:attribute>
+    </xs:complexType>
+
+    <xs:complexType name="OpenStackIdentityFQCNType">
```

Comment:
It might be nice to go ahead and split this out into it's own XSD that is imported in the examples/tests so that it more explicitly shows how this great capability can be extended.